src/sentry/shared_integrations/client/base.py

```diff
@@ -44,11 +45,13 @@ class BaseApiClient(TrackResponseMixin):
 
     def __init__(
         self,
+        integration_id: int,
```

Comment:
You'll need to make this optional or pass it for the other integrations that inherit from this